code/_onclick/hud/map_popups.dm

```diff
@@ -19,6 +19,17 @@
 /atom/movable/screen/proc/set_position(x, y, px = 0, py = 0)
 	if(assigned_map)
 		screen_loc = "[assigned_map]:[x]:[px],[y]:[py]"
+		ASYNC
+			// HACK: This fixes the character creator in 516 being small and relying on other byondui things (like cameras) to open in order to update and refresh.
+			// This also will fix the camera console screen being offset, Gateway, and admin pod panel.
+			// Adding 100 then setting it back seemed to do the trick!
```

Comment:
this is gonna look dumb as hell on the camera screen

Reply:
I tested all the screens and they looked fine. Cameras also suffer from the offset